src/test/java/org/reactome/release/dataexport/utilities/FTPFileUploaderTestUtils.java

```diff
@@ -18,66 +18,78 @@
 import java.util.Objects;
 import java.util.Properties;
 import java.util.stream.Collectors;
+
 import org.apache.commons.net.ftp.FTPClient;
 import org.mockito.Mockito;
 
 public class FTPFileUploaderTestUtils {
-	public static Properties getITTestPropertiesObject() {
-		final String realConfigPropertiesFileName = "real_config.properties";
-		try {
-			return getTestPropertiesObject(realConfigPropertiesFileName);
-		} catch (IOException | URISyntaxException e) {
-			throw new RuntimeException(
-				"ERROR!  To successfully run integration tests, a configuration file with real values for the "
-					+ "EuropePMC and NCBI server configuration values must be provided in this project with the file"
-					+ "path src/test/resources/" + realConfigPropertiesFileName + ".  For a sample configuration "
-					+ "file, see src/main/resources/sample_config.properties.", e
-			);
-		}
-	}
 
-	public static Properties getMockTestPropertiesObject() throws IOException, URISyntaxException {
-		return getTestPropertiesObject("mock_config.properties");
+	public static Properties getTestPropertiesObject() throws IOException, URISyntaxException {
+		return getITTestPropertiesObject() != null ? getITTestPropertiesObject() : getMockTestPropertiesObject();
 	}
 
-	public static Properties getTestPropertiesObject(String configResourceFileName)
-		throws IOException, URISyntaxException {
+	public static void createMockLocalFilesOutputDirectory() throws URISyntaxException, IOException {
+		Files.createDirectories(Paths.get(getMockLocalFilesOutputDirectory()));
+	}
 
-		URL urlToConfigResource = FTPFileUploaderTestUtils.class.getClassLoader().getResource(configResourceFileName);
-		if (urlToConfigResource == null) {
-			throw new IOException("Unable to find resource " + configResourceFileName);
+	public static void createFileIfDoesNotExist(Path file) throws IOException {
+		if (Files.notExists(file)) {
+			Files.createFile(file);
 		}
+	}
 
-		Properties props = new Properties();
-		props.load(new FileInputStream(urlToConfigResource.getPath()));
-		props.setProperty("outputDir", getMockOutputDirectory());
-		return props;
+	public static Path createTestUploadFile(Path directoryToCreateFile) throws IOException {
+		Files.createDirectories(directoryToCreateFile);
+
+		Path testUploadFilePath = getTestUploadFilePath(directoryToCreateFile);
+		createFileIfDoesNotExist(testUploadFilePath);
+		return testUploadFilePath;
 	}
 
-	public static String getMockOutputDirectory() throws URISyntaxException {
+	public static Path getTestUploadFilePath(Path directoryToCreateFile) {
+		final String testUploadFileName = "dummy.txt";
+
+		return directoryToCreateFile.resolve(testUploadFileName);
+	}
+
+	public static String getMockLocalFilesOutputDirectory() throws URISyntaxException {
```

Comment:
The name of this is weird. As far as I can tell, it returns a _real_ directory (or path to one) so it shouldn't have "mock" in the name. Maybe this needs renaming. When I see "getMockSomething..." I assume that a mock of type "Something" will be returned.

Reply:
Fair point - I'll go through and make sure all methods with "mock" are renamed unless they are actually creating a mock.